components/Card.tsx

```diff
@@ -0,0 +1,26 @@
+import { ComponentChildren, JSX } from 'preact';
+
+type CardProps = {
+  title: string;
+  content: string | JSX.Element;
+  link?: string;
+  link_text?: string;
+  children: ComponentChildren;
+};
```

Comment:
```suggestion
type CardProps = {
  title: string;
  content?: string | JSX.Element;
  link?: string;
  link_text?: string;
  children?: ComponentChildren;
};
```

There are some type errors for places using `Card` without `content` or `children` specified. Seems it can work fine without them, this type just needs to reflect that!